src/pages/saksoversikt/vilkår/Sats.tsx

```diff
@@ -25,14 +26,47 @@ const toBehandlingsinformasjonSats = (values: FormData): Nullable<Behandlingsinf
         return null;
     }
     return {
-        delerBolig: true,
+        delerBolig: values.delerSøkerBolig,
         delerBoligMed: values.delerBoligMedHvem,
         ektemakeEllerSamboerUførFlyktning: values.mottarEktemakeEllerSamboerSU,
         ektemakeEllerSamboerUnder67År: values.erEktemakeEllerSamboerUnder67,
         begrunnelse: null,
     };
 };
 
+const utledSats = (values: FormData) => {
+    if (values.delerSøkerBolig === null) {
+        return null;
+    }
+    if (!values.delerSøkerBolig) {
+        return FaktiskSats.Høy;
+    }
+    switch (values.delerBoligMedHvem) {
+        case null:
+            return null;
+        case DelerBoligMed.VOKSNE_BARN:
+            return FaktiskSats.Lav;
+        case DelerBoligMed.ANNEN_VOKSEN:
+            return FaktiskSats.Lav;
```

Comment:
Kanske man kan bruke fall-through för liknande utfall? exempelvis: 

```suggestion
        case DelerBoligMed.VOKSNE_BARN:
        case DelerBoligMed.ANNEN_VOKSEN:
            return FaktiskSats.Lav;
```